src/Components/Header.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react'
+import Form from './Form'
+
+class Header extends Component {
+    render() {
+        const {title} = this.props
+        return (
```

Comment:
We could also spread the rest of the props here `const { title, ...rest } = this.props` and pass {...rest} to the form component as {...this.props} will pass down all the props, but we dont need 'title' prop in the form